hackathon_site/dashboard/frontend/src/components/teamDetail/TeamCheckoutOrderTable/TeamCheckoutOrderTable.tsx

```diff
@@ -0,0 +1,227 @@
+                                                <img
+                                                    className={styles.itemImg}
+                                                    src={
+                                                        hardware[row.id]?.picture ??
+                                                        hardwareImagePlaceholder
+                                                    }
+                                                    alt={hardware[row.id]?.name}
+                                                />
+                                            </TableCell>
+                                            <TableCell>
+                                                {hardware[row.id]?.name}
+                                            </TableCell>
+                                            <TableCell align={"center"}>
+                                                <IconButton
+                                                    size={"small"}
+                                                    style={{
+                                                        marginLeft: "-65px",
+                                                    }}
+                                                >
+                                                    <Info />
+                                                </IconButton>
+                                            </TableCell>
+                                            <TableCell>
+                                                {`${
+                                                    hardware[row.id]?.quantity_available
+                                                }`}
+                                            </TableCell>
+                                            <TableCell>
+                                                <div
+                                                    style={{
+                                                        display: "flex",
+                                                        alignItems: "end",
+                                                    }}
+                                                >
+                                                    <Link
+                                                        underline="always"
+                                                        color="textPrimary"
+                                                        style={{ marginRight: "15px" }}
+                                                    >
+                                                        All
+                                                    </Link>
+                                                    <FormControl
+                                                        variant="standard"
+                                                        hiddenLabel={true}
+                                                    >
+                                                        <Formik
```

Comment:
You should make this Formik tag wrap the whole thing instead of using it within the loop. In this case, you are creating a form for every single hardware, which is not right.